### DISCUSSION TITLE
tankman miss offsets in txt file?

### DISCUSSION THREAD

**Discussion by AbnormalPoof** [FunkinCrew/Funkin#1769] [Q&A] [answered]
i found some things relating to miss poses when looking at tankmans offset txt file
![Screenshot 2021-09-19 10 01 20 PM](https://user-images.githubusercontent.com/86753001/134096365-f97a6f44-6774-45f6-8f10-446202eaa6ef.png)
Is this a mistake or just a playable character leftover

**Comment by DotWith** [comment]
Yea if you look in the week7 code its copied and pasted code from pico, and so is the fla :trollface:. This could mean theres miss animations for him. Plus I think it could be like old ps3 games where you beat the character to play as them. Lets not forget about the thing ninjamuffin99 is adding `Multiplayer`. soooooooo yeaa. Sorry, for bad grammar just got home from high school.
![image](https://user-images.githubusercontent.com/52358442/134228111-6af6afc7-d4e6-4c4c-bbed-fd91c851b447.png)

**Reply by just-pork** [reply]
The code for Tankman's Missing Note Animations are worded differently for some reason as it's "Tankman Note Left MISS" and not something like "Pico NOTE LEFT miss"
We already know Tankman has "missing note animations" as concept art. A lot of stuff from Tankman is missing though like his cutscene sprites for Ugh and Stress. 
Another cool little small fact is that the [first release](http://uploads.ungrounded.net/alternate/1528000/1528775_alternate_113347_r86.zip/) of Week 7 on Newgrounds has Week 7's Story Mode broken, looking through the Funkin.js it shows that the code for the original native cutscene for Ugh was still in place but with the files for Tankman missing it just crashes.

**Reply by DotWith** [reply]
That unused cutscenes for week7 don't work because of some key shit on the animations 😢

**Reply by just-pork** [reply]
For some reason, the sprites for Guns intro and the Speakers Animation for Stress are still in the files but not the files for Ugh which is why it just doesn't work, for some reason too Guns and Stress both have their code to be video cutscenes but not Ugh's.

**Reply by AbnormalPoof** [reply]
wait so ugh's sprites were in the initial release?

**Reply by just-pork** [reply]
Unfortunately. No
![Screenshot 2021-09-22 7 37 55 AM](https://user-images.githubusercontent.com/89614860/134345635-cc4b93e7-e4e2-45aa-ab24-cb080c81db4a.png)
What does exist however is the code for Ugh's cutscene. It makes mention of the files used. You could possibly try and make your own sprites for it and see if it works with the game. I haven't done it as of yet.

**Reply by DotWith** [reply]
> Unfortunately. No
> 
> ![Screenshot 2021-09-22 7 37 55 AM](https://user-images.githubusercontent.com/89614860/134345635-cc4b93e7-e4e2-45aa-ab24-cb080c81db4a.png)
> 
> What does exist however is the code for Ugh's cutscene. It makes mention of the files used. You could possibly try and make your own sprites for it and see if it works with the game. I haven't done it as of yet.
> 
> 

Ok, when I get home I'll try and add the unused animations.

**Reply by AbnormalPoof** [reply]
i couldve sworn ive seen a cutscenestuff folder
even if its like from a fuckin bootleg or shit

**Reply by DotWith** [reply]
> i couldve sworn ive seen a cutscenestuff folder
> 
> even if its like from a fuckin bootleg or shit

That folder is private it was shown on ninjamuffins stream.

**Reply by just-pork** [reply]
I wish you luck trying to decode this as Funkin.js really simplifies the code and makes my brain even smoother though if you get this working, you can probably get Guns' intro working.

**Reply by DotWith** [reply]
> I wish you luck trying to decode this as Funkin.js really simplifies the code and makes my brain even smoother though if you get this working, you can probably get Guns' intro working.

Who said I've haven't already

**Reply by kinoko87** [reply]
Interesting, I thought they played a video, not make like a bunch of animations to tankman, etc

**Reply by DotWith** [reply]
> Interesting, I thought they played a video, not make like a bunch of animations to tankman, etc

They couldn't get the animations working so they used a video.

**Reply by AbnormalPoof** [reply]
> Who said I've haven't already

i dont see the guns intro lmao

**Reply by DotWith** [reply]
> > Who said I've haven't already
> 
> 
> 
> i dont see the guns intro lmao

I didn't add that yet as a video, but the newest shit on the net-test branch

**Reply by just-pork** [reply]
The Funkin Decompilation

**Reply by AbnormalPoof** [reply]
I managed to add my own sprites and the ugh cutscene loaded

**Comment by ninjamuffin99** [comment, marked as answer]
yeah, same reason why pico has miss animation stuff. they r both gonna be playable eventually

**Reply by AbnormalPoof** [reply]
I thought it was a pico copy paste considering he also has it lol
